### PR TITLE
Billing

### PR DESCRIPTION
**Pull request by wcpines**
Closes https://github.com/aptible/support/issues/85

### CODE REVIEW

File: source/topics/account/how-to-manage-billing.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - The estimated monthly cost of your current configuration
+    - Your account's current balance
+
+4. To pay by ACH, send a support ticket to support@aptible.com with the following bank information:
+
+    - Account owner name 
+    - Routing Number
+    - Account Number
+
+5. To change the recipient of monthly invoices (automatically sent at time of payment), please [contact support](support@aptible.com).
```
Comment (fancyremarker) [inline source/topics/account/how-to-manage-billing.md]: That's not a valid link. You want `mailto:support@aptible.com`. Side note: let's remember to change all these mailto links to point to the support request form.

**Comment by fancyremarker** [comment]
One minor change, then this is merge-ready.

**Comment by wcpines** [comment]
Changing the emails addresses down the line is easy.

**Comment by fancyremarker** [comment]
Merging, thanks.

**Pull request merged**